docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,176 @@
+
+   ```bash
+   platform mount:download --mount {{< variable "MOUNT_PATH" >}} --target ./{{< variable "LOCAL_FOLDER" >}}
+   ```
+
+<--->
+
+---
+title=Using SSH
+file=none
+highlight=false
+---
+
+See how to transfer files [using `scp`](../development/file-transfer.md#scp) or [`rsync`](../development/file-transfer.md#rsync).
+
+{{< /codetabs >}}
+
+For more examples, see how to [transfer files to and from a built app](../development/file-transfer.md).
+
+## 3. Download data from services
 
 The mechanism for downloading from each service (such as your database) varies.
-For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.
 
-To download data from persistent services ([MySQL](../add-services/mysql/_index.md), [PostgreSQL](../add-services/postgresql.md), [MongoDB](../add-services/mongodb.md), or [InfluxDB](../add-services/influxdb.md)), see each service's page for instructions.
+For services designed to hold non-persistent data, such as Redis or Solr,
+it's generally not necessary to download data as it can be rebuilt from the primary data store.
+
+For services designed to hold persistent data, see each service's page for instructions:
+
+- [MySQL](../add-services/mysql/_index.md#exporting-data)
+- [PostgreSQL](../add-services/postgresql.md#exporting-data)
+- [MongoDB](../add-services/mongodb.md#exporting-data)
+- [InfluxDB](../add-services/influxdb.md#exporting-data)
+
+## 4. Get the environment variables
+
+Environment variables can contain critical information such as tokens or additional configuration options for your app.
+
+Environment variables beginning with:
 
-## Get the environment variables
+- `env:` are exposed [as Unix environment variables](../development/variables/_index.md#top-level-environment-variables).
+- `php:` are interpreted [as `php.ini` directives](../development/variables/_index.md#php-specific-variables).
 
-If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.
+All other variables are [part of `$PLATFORM_VARIABLES`](../development/variables/use-variables.md#use-platformsh-provided-variables).
 
-As stated in the Console, several possibilities exist for the environment variables.
+To backup your environment variables:
 
-* Variables beginning with `env:` are exposed as Unix environment variables
-* Variables beginning with `php:` are interpreted as `php.ini` directives.
-* All other variables are part of the environment `PLATFORM_VARIABLES` variable
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. Get the variable's values, by running:
+
+   ```bash
+   platform ssh --project {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT" >}} -- 'echo $PLATFORM_VARIABLES | base64 -d | jq'
+   ```
+
+2. Optional: To get all the environment variable values, run:
```

Comment:
❓ Is this really an optional second step or just an alternative to the first step? (=> Honest question, are users really likely to do number 1 **then** number 2, or rather number 1 **or** number 2?). 
If the latter is true, instead of having the optional 2nd step, I might add a note after the first command. Something like: "Note that you can also get all the environment variable values with + command". And then move on to the 3rd step. What do you think?